examples/list.c

```diff
@@ -195,25 +190,19 @@ void step_main(struct ssm_act *act) {
 }
 
 void ssm_program_init(void) {
-  struct ssm_object *obj = ssm_new(List_size, Nil);
-  list = ssm_from_obj(obj);
-  obj = ssm_new(List_size, Cons);
-  obj->payload[0] = ssm_marshal(3);
-  obj->payload[1] = list;
-  list = ssm_from_obj(obj);
-  obj = ssm_new(List_size, Cons);
-  obj->payload[0] = ssm_marshal(2);
-  obj->payload[1] = list;
-  list = ssm_from_obj(obj);
-  obj = ssm_new(List_size, Cons);
-  obj->payload[0] = ssm_marshal(1);
-  obj->payload[1] = list;
-  list = ssm_from_obj(obj);
-  ssm_act_t *act =
-      ssm_enter_main(&ssm_top_parent, SSM_ROOT_PRIORITY, SSM_ROOT_DEPTH);
-  ssm_activate(act);
+  ssm_value_t v;
+  v = ssm_marshal(Nil);
+  list = v;
+  int i = ssm_init_args && ssm_init_args[0] ? atoi(ssm_init_args[0]) : 3;
+  for (; i >= 1; i--) {
```

Comment:
Added: the size of the initial list in the list example is now parameterized by the command line argument.